data/themes/darktable.css

```diff
@@ -1016,33 +1043,33 @@ notebook tabs,
 }
 
 /* set sizes of "buttons" in modules and blending tabs ; and guides lines too */
-#modules-tabs #dt-toggle-button
+#modules-tabs .dt_module_btn
 {
   min-height: 2em;
 }
 
 #blending-tabs .dt_module_btn,
-#guides-line #dt-toggle-button
+#guides-line .dt_module_btn
 {
   min-height: 1.3em;
   padding: 0.28em;
 }
 
-#modules-tabs #dt-toggle-button,   /* This is needed separately to set a minimal width if panel is really narrow */
-#blending-tabs #dt-toggle-button,
-#guides-line #dt-toggle-button
+#modules-tabs .dt_module_btn,   /* This is needed separately to set a minimal width if panel is really narrow */
+#blending-tabs .dt_module_btn,
+#guides-line .dt_module_btn
 {
   min-width: 1.5em;
 }
 
 /* reduce and align now hamburger menu of modules and blending tabs */
-#blending-tabs #dt-button,
-#modules-tabs #dt-button
+#blending-tabs .dt_module_btn,
+#modules-tabs .dt_module_btn
```

Comment:
This make hamburger menu too big on both parts. With lines just above, it's needed and wanted to keep difference between dt-toggle-button and dt-button